### PR TITLE
create sabe.md

### PR DESCRIPTION
**Pull request by ghost**
Fixes #issue_number

### Description
Description of changes made.

### Raw.Githack preview link
raw.githack link to page(s) changed.

e.g. https://raw.githack.com/sabe-lab/sabe-lab.github.io/sabe-new-branch/#!pages/vi/profiles/sabe-lab.md

### CODE REVIEW

**Comment by phamduchongan93** [comment]
@sabe-lab Hello and welcome. Very professional work :). We will review your PR soon.

**Comment by irisb1701** [comment]
Welcome @sabe-lab, nice work

**Review by irisb1701 — APPROVED** [review]

File: pages/vi/profiles/sabe-lab.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+
+```php
+  function main(){
+    return "Nice to meet you !";
+   }
+```
+
+
+![ninja](https://raw.githack.com/sabe-lab/sabe-lab.github.io/sabe-new-branch/pages/vi/profiles/images/ninja.jpg)
```
Comment (dogi) [inline pages/vi/profiles/sabe-lab.md]: please make a relative link

**Review by dogi — CHANGES_REQUESTED** [review]
@sabe-lab good job

**Review by kjong — APPROVED** [review]

File: pages/vi/profiles/sabe-lab.md (hunk 1 above)
Comment (dogi) [reply pages/vi/profiles/sabe-lab.md]: `ninja
 `

**Comment by sabe-lab** [comment]
pages/vi/profiles/sabe-lab.md

**Review by dogi — APPROVED** [review]

**Pull request merged**